sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering-validation.models.ts

```diff
@@ -46,7 +46,8 @@ import {
   OFFERING_STUDY_BREAK_MAX_DAYS,
   OFFERING_STUDY_BREAK_MIN_DAYS,
   OFFERING_STUDY_PERIOD_MAX_DAYS,
-  OFFERING_STUDY_PERIOD_MIN_DAYS,
+  FT_OFFERING_STUDY_PERIOD_MIN_DAYS,
```

Comment:
Minor but I guess that we usually go for the extended names.
I would prefer to have those named as:
`OFFERING_STUDY_PERIOD_MIN_DAYS_FULL_TIME`
`OFFERING_STUDY_PERIOD_MIN_DAYS_PART_TIME`

![image](https://github.com/bcgov/SIMS/assets/61259237/3f93cc1d-27f2-4109-a9cb-cfdf1db10392)

![image](https://github.com/bcgov/SIMS/assets/61259237/1d9d6b28-72ef-439a-bf51-58394303eabb)